doc/changes/changes_0.1.2.md

```diff
@@ -0,0 +1,37 @@
+# Virtual Schema for Snowflake 0.1.2, released 2025-02-10
+
+Code name: Fixed vulnerabilities in net.snowflake:snowflake-jdbc:jar:3.20.0:compile
+
+## Summary
+
+This release fixes vulnerabilities [CVE-2025-24789](https://github.com/advisories/GHSA-7hpq-3g6w-pvhf)
+and [CVE-2025-24790](https://github.com/advisories/GHSA-33g6-495w-v8j2) in dependency `net.snowflake:snowflake-jdbc:jar:3.20.0:runtime`
```

Comment:
Note for the future: You don't need to create links for CVE numbers. GitHub automatically creates links.